plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/session/SessionStateTypes.kt

```diff
@@ -4,13 +4,15 @@
 package software.aws.toolkits.jetbrains.services.amazonqFeatureDev.session
 
 import com.fasterxml.jackson.annotation.JsonValue
+import org.gradle.tooling.CancellationTokenSource
```

Comment:
I think if they don't have gradle plugin installed it can't even run currently

<img width="1728" alt="Screenshot 2024-10-23 at 8 03 35 AM" src="https://github.com/user-attachments/assets/c57154d2-59bb-4972-939d-7dfb2e462f7d">

Reply:
Removed gradle dependency